common/djangoapps/student/views.py

```diff
@@ -653,66 +730,142 @@ def dashboard(request):
             enrollment.course_id
         )
     )
-
     enrolled_courses_either_paid = frozenset(
         enrollment.course_id for enrollment in course_enrollments
         if enrollment.is_paid_course()
     )
 
-    # If there are *any* denied reverifications that have not been toggled off,
-    # we'll display the banner
-    denied_banner = any(item.display for item in reverifications["denied"])
+    courses_context = []
+    for course in course_enrollments:
+        course_context = {
+            "course_id": course.id,
+            "course_key": unicode(course.course_id),
+            "course_number": course.course_overview.number,
+            "course_name": course.course_overview.display_name_with_default,
+            "course_image_url": course.course_overview.image_urls['small'],
```

Comment:
If you don't mind, could you just send all the image_urls -- i.e. pass in the whole dict to the template, and let the template decide which resolutions it will use? I'm thinking mostly from the standpoint of theming flexibility and things like srcset for retina display.